org-roam.el

```diff
@@ -529,21 +528,19 @@ Valid states are 'visible, 'exists and 'none."
   "Disable org-roam updating for file.
 
 1. Remove hooks for updating the cache, and the org-roam buffer."
-  (remove-hook 'post-command-hook #'org-roam--maybe-update-buffer)
-  (remove-hook 'after-save-hook #'org-roam--update-cache))
+  (remove-hook 'post-command-hook #'org-roam--maybe-update-buffer t)
+  (remove-hook 'after-save-hook #'org-roam--update-cache t))
 
 (cl-defun org-roam--maybe-update-buffer (&key redisplay)
   "Update `org-roam-buffer' with the necessary information.
 This needs to be quick/infrequent, because this is run at
 `post-command-hook'."
-  (with-current-buffer (window-buffer)
-    (when (and (get-buffer org-roam-buffer)
```

Comment:
shouldn't we still check if the `*org-roam*` buffer exists?

Reply:
Good catch.